### PR TITLE
fix(python): Python props were ugly and not very pythonic

### PR DESCRIPTION
**Pull request by HBobertz**
I realized I didn't need a ternary operator to default these values and that the second positional argument of kwargs.get is a built in default value

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/synthesizer/python/mod.rs [hunk 1]
```diff
@@ -18,6 +18,8 @@ use super::Synthesizer;
 
 const INDENT: Cow<'static, str> = Cow::Borrowed("  ");
 
+// reserved python keywords as of 3.13 (https://docs.python.org/3/reference/lexical_analysis.html#keywords)
+// if we happen to name a module with one of these keywords, we need to prepend 'aws_' to avoid a name conflict
```
Comment (HBobertz) [inline src/synthesizer/python/mod.rs:22]: decided to just throw in this comment while I was at it. LMK if you want me to put this in it's own pr

File: src/synthesizer/python/mod.rs [hunk 2]
```diff
@@ -85,7 +89,6 @@ impl Synthesizer for Python {
             });
             for param in have_default_or_special_type_params {
                 let name = &param.name;
-                // example: AWS::EC2::Image::Id, List<AWS::EC2::VPC::Id>, AWS::SSM::Parameter::Value<List<String>>
```
Comment (HBobertz) [inline src/synthesizer/python/mod.rs:88]: this honestly isn't very helpful, I just decided to remove it

Comment (TheRealAmazonKendra) [reply src/synthesizer/python/mod.rs:88]: I've actually found this comment helpful. Can we please leave it in?

Comment (HBobertz) [reply src/synthesizer/python/mod.rs]: oh sure, yeah re-added it

**Review by TheRealAmazonKendra — APPROVED** [review]
